client/packages/coldchain/src/Monitoring/ListView/TemperatureChart/useTemperatureChartData.ts

```diff
@@ -9,10 +9,21 @@ const MAX_DATA_POINTS = 30;
 
 export const useTemperatureChartData = () => {
   const theme = useTheme();
-  const {
-    queryParams,
-  } = useUrlQueryParams({
-    initialSort: {key: 'datetime', dir: 'desc'},
+  const { filter, queryParams } = useUrlQueryParams({
+    initialSort: { key: 'datetime', dir: 'desc' },
+    filters: [
+      { key: 'datetime', condition: 'between' },
```

Comment:
Should this be consolidated with ListView ?

https://github.com/msupply-foundation/open-msupply/blob/c57b07040c13979ca5ffc37af229bde7620b90f8/client/packages/coldchain/src/Monitoring/ListView/TemperatureLog/TemperatureLogList.tsx#L25-L41

Reply:
yeah, I think it should - the breach filters differ slightly, but the log and chart ones are exactly the same.